sdk/synapse/Azure.Analytics.Synapse.ManagedPrivateEndpoints/tests/ManagedPrivateEndpointsClientLiveTests.cs

```diff
@@ -27,7 +27,7 @@ public ManagedPrivateEndpointsClientLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
-        [Test]
+        //[Test] - https://github.com/Azure/azure-sdk-for-net/issues/17455
```

Comment:
suggestion: I'd leave the Test attributes and add an Ignore like
`[Ignore("https://github.com/Azure/azure-sdk-for-net/issues/17455")]`

Reply:
Oh, my last team banned IgnoreAttribute at one point. If that's usable I'll change to that.

Reply:
oh! curious, do u know why?